docs/src/hybrid_search.md

```diff
@@ -130,6 +130,55 @@ Arguments
     Only returns `_relevance_score`. Does not support `return_score = "all"`.
 
 
+### ColBERT Reranker
+This reranker uses the ColBERT model to combine the results of semantic and full-text search. You can use it by passing `ColbertrReranker()` to the `rerank()` method. 
```

Comment:
Somewhere here you should a bit more explanation that:
1. The ColbertReranker needs the actual document rather than purely based on scores 
2. By default it uses the "text" column but if it's different in your table you can specify it in the ColbertReranker constructor